README.adoc

```diff
@@ -136,7 +131,7 @@ also be application-scoped, request scope is short-lived and is therefore ideal
 Add `@Inject InventoryManager manager;` annotation on the class
 `src/main/java/io/openliberty/guides/inventory/InventoryResource.java`.
 
-Your completed class looks like this:
+Your completed class appears as it does in the following example:
 
```

Comment:
introduce the list--better for non-sighted users that rely on a screen reader.

Reply:
Ok.